webview/src/experiments/components/table/styles.module.scss

```diff
@@ -298,6 +283,8 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
   }
 
   .bodyRow {
+    border-bottom: $row-border;
```

Comment:
I used our original border color for the row lines (which isn't black compared to the Figma). Do we want to use more a black border color to better match the Figma?

Figma:

<img width="620" alt="image" src="https://user-images.githubusercontent.com/43496356/182492232-a11616d5-0fd1-4b8a-a365-fdca652bdcd4.png">

Current: 

<img width="361" alt="image" src="https://user-images.githubusercontent.com/43496356/182492282-ba73a69d-4b27-491b-9b5f-9109db7536e9.png">

Reply:
Obviously, Figma is the guide. We can jump on a call tomorrow to discuss the small details. However, the base theme needs to look like in Figma (including 0.5px borders). Thanks, @julieg18!

Reply:
What does this look like on a high contrast theme? If we do not offer high contrast for at least these themes, it's not very good on our part.

Reply:
>  However, the base theme needs to look like in Figma (including 0.5px borders, BG color).

Updated the border width to `0.5`! As for border colors, I looked into an alternative color that was more like black in the dark theme but couldn't find a color variable darker than the current table background color. I did look into using just `#000` but that didn't look good in some of the darker themes :(

> What does this look like on a high contrast theme? If we do not offer high contrast for at least these themes, it's not very good on our part.

On storybook, contrast themes looks good to me... 

<img width="476" alt="image" src="https://user-images.githubusercontent.com/43496356/182636815-b4a869cc-c2ae-4c43-aa01-605d85508b6b.png">

<img width="415" alt="image" src="https://user-images.githubusercontent.com/43496356/182636996-fbbca00e-82e3-4906-ab80-373abfe24775.png">